src/plugins/data/server/search/strategies/esql_search/esql_search_strategy.ts

```diff
@@ -41,8 +54,11 @@ export const esqlSearchStrategyProvider = (
             },
           },
           {
-            signal: abortSignal,
+            signal: abortController.signal,
             meta: true,
+            // we don't want the ES client to retry (default value is 3)
+            maxRetries: 0,
+            requestTimeout: ES_TIMEOUT_IN_MS,
```

Comment:
I think if we're already manually timing out (using the `AbortController`), we don't need to set this `requestTimeout` value here. My concern is that if it has already been overridden to something lower than 2 mins, then we will be overriding it here to actually allow longer-than-configured queries.

Reply:
You are right Lukas, I just forgot to remove it :)